Makefile

```diff
@@ -128,6 +128,10 @@ _login_to_docker:
 	sudo docker login -u $(shell oc whoami) -p $(shell oc whoami -t) $(_DOCKER_REGISTRY)
 .PHONY: _login_to_docker
 
+_login_to_openshift:
+	oc login -u $(_OPENSHIFT_USERNAME) -p $(_OPENSHIFT_PASSWORD)
+.PHONY: _login_to_openshift
+
```

Comment:
Hmm, there's already a target called login-to-openshift. How did you run into this issue? You probably didn't call test-ci, right?

Reply:
It was removed at some point. This error presents itself when running the services manually, e.g. `make stop-openshift start-openshift-with-catalog build-image push-image-to-local-openshift install-templates test-caching-service-manually`

Reply:
OK. Then you could probably re-use the login-to-openshift (without the underscore at the beginning) and add it only to push-image-to-local-openshift ?

Reply:
There isn't a `login-to-openshift` on master anymore :smile: 

https://github.com/jboss-container-images/jboss-dataservices-image/blob/dataservices-dev/Makefile

Reply:
:-D